massa-execution-worker/src/interface_impl.rs

```diff
@@ -1066,8 +1070,11 @@ impl Interface for InterfaceImpl {
     /// # Arguments:
     /// data: the bytes_array data that is the payload of the event
     fn generate_event_wasmv1(&self, data: Vec<u8>) -> Result<()> {
```

Comment:
we should pass data as &[u8] here, we're probably doing a useless copy

Reply:
Opened a follow up issue #4298